pkgs/by-name/at/atlauncher/package.nix

```diff
@@ -56,4 +69,4 @@ stdenv.mkDerivation (finalAttrs: {
     platforms = platforms.all;
     sourceProvenance = [ sourceTypes.binaryBytecode ];
   };
-})
+}
```

Comment:
```suggestion
})
```

Reply:
finalAttrs should be used in place of `rec` wherever possible. It makes overriding more effective and in your case, allows you to remove the `let` binding that you made, see: https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes